app/pages/reset-password/reset-password.spec.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import ResetPasswordPage from './reset-password';
+
+const user = {
+  display_name: 'Test User'
+};
+
+const location = {
+  query: {
+    reset_password_token: 'test'
+  }
+};
+
+describe('ResetPasswordPage', () => {
+  let wrapper;
+
+  before(() => {
+    wrapper = shallow(<ResetPasswordPage />, { context: { router: { push: () => {} }}});
+  });
+
+  it('should render without crashing', () => {
+    const ResetPasswordPageContainer = wrapper.find('div.centered-grid');
+    assert.equal(ResetPasswordPageContainer.length, 1);
+  });
+
+  describe('SubmitEmailForm', () => {
+    it('renders without crashing', () => {
+      assert.equal(wrapper.find('SubmitEmailForm').length, 1);
+    });
+
+    it('renders with no user if none is passed in', () => {
+      assert.equal(wrapper.find('SubmitEmailForm').nodes[0].props.user, null)
```

Comment:
If you find yourself trying to use `node[i]` in your test, you probably either need to use a better selector to find what you're trying to test or you are trying to test something in a child component and this test should be written for a set a tests for that child component.